### PR TITLE
Remove ThreeTenABP library

### PR DESCRIPTION
**Pull request by AndrKriv**

### CODE REVIEW

File: app/src/main/java/com/innowisegroup/datetimepicker/CustomViewPager.kt [hunk 1]
```diff
@@ -6,21 +6,21 @@ import android.view.MotionEvent
...
     private var enable = true
-    
+
     override fun onTouchEvent(event: MotionEvent): Boolean =
-        if (enable) {
-            super.onTouchEvent(event)
-        } else false
+            if (enable) {
+                super.onTouchEvent(event)
+            } else false
```
Comment (dmorskoi) [inline app/src/main/java/com/innowisegroup/datetimepicker/CustomViewPager.kt]: Here actually you can merge it to one-string expression, cause they are simple
`    if (enable) super.onTouchEvent(event) else false`

The same in the code below

File: app/src/main/java/com/innowisegroup/datetimepicker/LocalTime.kt [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+class LocalTime() : Serializable {
+    constructor(hour: Int, minute: Int) : this() {
+        this.hour = hour
+        this.minute = minute
+    }
+
+    private var hour: Int? = null
+    private var minute: Int? = null
+
+    private var calendar: Calendar = Calendar.getInstance()
```
Comment (dmorskoi) [inline app/src/main/java/com/innowisegroup/datetimepicker/LocalTime.kt]: why var?

File: app/src/main/java/com/innowisegroup/datetimepicker/LocalDate.kt [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        this.day = day
+        this.month = month
+        this.year = year
+    }
+
+    private var day: Int? = null
+    private var month: Int? = null
+    private var year: Int? = null
+
+    private var calendar: Calendar = Calendar.getInstance()
```
Comment (dmorskoi) [inline app/src/main/java/com/innowisegroup/datetimepicker/LocalDate.kt]: why var?

File: app/src/main/java/com/innowisegroup/datetimepicker/Month.kt [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    MARCH,
+    APRIL,
+    MAY,
+    JUNE,
+    JULY,
+    AUGUST,
+    SEPTEMBER,
+    OCTOBER,
+    NOVEMBER,
+    DECEMBER;
```
Comment (Bwonner) [inline app/src/main/java/com/innowisegroup/datetimepicker/Month.kt:15]: Redundant " ; "

**Review by dmorskoi — APPROVED** [review]

**Pull request merged**